packages/kokoas-client/src/pages/projSearch/sections/result/DownloadResult.tsx

```diff
@@ -27,8 +27,13 @@ export const DownloadResult = ({
         '顧客名（連名）': d.custNames,
         電話番号: d.tel,
         工事名: d.projName,
+        顧客郵便番号: d.custPostalCode,
         顧客住所: d.custAddress,
-        工事住所: d.projAddress,
+
+        // K246 工事確定住所が入力されている工事は、工事確定住所を工事住所として表示してほしい
+        工事郵便番号: d.projAddress ? d.projPostalCode : d.projPostalCodeConfirmed,
+        工事住所: d.projAddress || d.projAddressConfirmed,
```

Comment:
```suggestion
        // K246 工事確定住所が入力されている工事は、工事確定住所を工事住所として表示してほしい
        工事郵便番号: d.projPostalCodeConfirmed? d.projPostalCodeConfirmed : d.projPostalCode,
        工事住所: d.projAddressConfirmed || d.projAddress,
```
上記コメントと同様です。
認識違いがあれば教えてください。

Reply:
上記と同様でそのまま適用させて頂きました。

ありがとうございます。